evaluator/autoware_planning_evaluator/src/metrics/deviation_metrics.cpp

```diff
@@ -45,6 +45,24 @@ Accumulator<double> calcLateralDeviation(const Trajectory & ref, const Trajector
   return stat;
 }
 
+Accumulator<double> calcLateralTrajectoryDisplacement(
+  const Trajectory & prev, const Trajectory & traj, const Pose & ego_pose)
+{
+  Accumulator<double> stat;
+
+  if (prev.points.empty() || traj.points.empty()) {
+    return stat;
+  }
+
+  const auto prev_lateral_deviation =
+    autoware::motion_utils::calcLateralOffset(prev.points, ego_pose.position);
+  const auto traj_lateral_deviation =
+    autoware::motion_utils::calcLateralOffset(traj.points, ego_pose.position);
+  const auto lateral_trajectory_displacement = traj_lateral_deviation - prev_lateral_deviation;
```

Comment:
- use std::value
- we want calculate lateral distance between nearest point in current and previous